### ISSUE TITLE
Rounded corners while maximized

### ISSUE THREAD

**Issue by eshate** [witalihirsch/Mono-firefox-theme#10]
**Thanks for this theme, I am using full GTK with pleasure!**

I have this problem though - window corners are still rounded when maximized. 
I am using "rounded corners" extension for gnome-shell, but the problem persists even when it's not working. It has to be something in CSS, but I really cannot find this selector. Can you put some light on what could I do with this?

This white space on the left - it's really tiny, but it's there:

![Screenshot from 2022-12-27 04-41-08](https://user-images.githubusercontent.com/74292287/209607813-f2ad66ce-527a-4a1d-adc4-719033aab7cb.png)

**Comment by witalihirsch** [comment]
could it still be a issue with some extension? I don't have that problem
![image](https://user-images.githubusercontent.com/73841541/209769445-df6a7788-484e-45ff-8e72-5c5a92ee9677.png)

**Comment by ChildishGiant** [comment]
I also don't have this issue, are you on the latest version?

**Comment by eshate** [comment]
Yes, I am on the last version - both firefox and global Mono GTheme. 
I've disabled all extensions and did `alt+F2+r` and the problem is still there. Even more - I found that well-known glitch with rounded corners when not-maximized (_screen attached_). 

**I assume that problem is on my side**, though. Something messed up, so I will try to manage it somehow. My proposition is to leave this issue **open for few days** - maybe someone would have an idea; and then I will close it. It's that okay for you? 

Again, thanks for the theme - it's makin my minimalistic aesthetic feel good! 

![Screenshot from 2022-12-28 18-12-07](https://user-images.githubusercontent.com/74292287/209849130-328b09b9-8ef0-4560-8c37-ad896838deb9.png)

**Comment by witalihirsch** [comment]
did you find the reason why this is happening?

**Comment by eshate** [comment]
Sorry for not responding faster, had some stuff on my head. 
I am not hundred percent sure, but I found some leftovers after _Firefox Gnome Theme_ which was also "installable". I am not used to userChrome which has to be "installed", so I'm not sure what leftovers it can leave (I've just deleted chrome folder, maybe it was not enough). But yeah, a messy userChrome - definetely on my side. 

Now I've back to classic theme though, just because this CSS fun was distracting me from work. But I'll install Mono soon with refreshed state.

Sorry to bother you, I'm closing this issue.Especially that I have one more in my pocket. If there is moral for someone who have similar problems and reading it - clean your chrome folder, that's always a good thing.
